infrastructure/tooling/templates/k8s/deployment.yaml

```diff
@@ -15,6 +15,10 @@ spec:
     spec:
       serviceAccountName: ${projectName}
       imagePullSecrets: IMAGE_PULL_SECRETS_STRING
+      # allow server to terminate gracefully while still serving existing connections
+      # new connections would be rejected as server will be closed
+      # and under normal conditions pod will stop faster than this timeout
+      terminationGracePeriodSeconds: 120
```

Comment:
This available option makes it configurable, but defaults to 120.

https://github.com/taskcluster/taskcluster/pull/6717/files#diff-644aef6929a112d41db773f1869a6699ac4ea3d90011ecacec29fb3c6afa0ae0R31

Reply:
so the default there is 30-45s (based on different versions or sources) which might be just fine also
I just increased it to give all existing connections enough time to get their responses and disconnect. 

Most important part is that server doesn't just kills itself when kubernetes says "I'm going to terminate you now.. are you gonna do something about it?" and so far we were like "meh, sure, whatever, just kill me.."
So now, instead of just accepting the fate and dying without saying a word, service will say "HOLD ON RIGHT THERE MR KUBE! I have some business to finish.. I will `server.close()` to avoid accepting new connections, but I will wait a bit for my existing customers to be served.. " And that's when timer starts to tick those 120s .. 

There is a chance this will contradict to the `KEEP_ALIVE_TIMEOUT_SECONDS` we introduced before, but! if someone keeps open connection, we don't have any calls that (theoretically) should take longer than 20s (claim work) .. so if such client comes back for another call, server would say "Sorry, we are closing, be quick"

I think it should be a huge improvement already